templates/ib/swift4.stencil

```diff
@@ -26,7 +26,7 @@ import {{module}}
   }
 }
 
-{{accessModifier}} struct SceneType<T: Any> {
+{{accessModifier}} struct SceneType<T: {% if isAppKit %}Any{% else %}UIViewController{% endif %}> {
```

Comment:
I wonder if it wouldn't be more idiomatic to just use `SceneType<T>` rather than `SceneType<T: Any>` for AppKit?